src/test/resources/exampleTestConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dataNodeAddress": "192.168.0.1",
+  "managementNodeAddress": "192.168.0.2",
+  "sshPort": 22,
+  "sshKey": "./exasol_cluster_ssh_key",
```

Comment:
See above: path to ssh key.

Reply:
it's from another repo and alreaddy released. Not sure if its woth a change